### ISSUE TITLE
Implementation details for training CoverNet

### ISSUE THREAD

**Issue by hobincar** [nutonomy/nuscenes-devkit#543]
Hi, thanks for releasing the code for CoverNet.
As the training code is not available, I'm trying to implement it myself. The learning rate and its decay schedule are written in the paper, but the other details are missing. Can you provide some additional information such as how many epochs it takes to converge, the batch size, the optimizer, etc.?

**Comment by emwolff** [comment]
@hobincar Sure. We used the following values:
Batch size: 64
Epochs: 20-30 for nuScenes 6 sec. Convergence was around 9.
Optimizer: SGD with momentum of 0.9 and weight decay of 5e-4.
